django/db/models/query.py

```diff
@@ -485,8 +485,7 @@ def update_or_create(self, defaults=None, **kwargs):
         for k, v in six.iteritems(defaults):
             setattr(obj, k, v)
 
-        with transaction.atomic(using=self.db, savepoint=False):
-            obj.save(using=self.db)
+        obj.save(using=self.db)
```

Comment:
This may affect query counts tested by `assertNumQueries` in `TransactionTestCase` which should be mentioned in the release notes.